editquality/feature_lists/huwiki.py

```diff
@@ -17,7 +17,8 @@
     hungarian.informals.revision.diff.match_delta_decrease,
     hungarian.informals.revision.diff.match_prop_delta_sum,
     hungarian.informals.revision.diff.match_prop_delta_increase,
-    hungarian.informals.revision.diff.match_prop_delta_decrease
+    hungarian.informals.revision.diff.match_prop_delta_decrease,
+    informals_no_ha = english.informals.excluding(["ha"])
```

Comment:
This will end up as syntax error, please move it outside of definition of the list.